components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1115,9 +1115,10 @@ void RewardsServiceImpl::OnPublisherActivityInfoLoaded(
     ledger::PublisherInfoCallback callback,
     uint32_t result,
     const std::string& info_json) {
-  auto publisher = std::make_unique<ledger::PublisherInfo>();
+  std::unique_ptr<ledger::PublisherInfo> publisher;
 
   if (!info_json.empty()) {
+    publisher.reset(new ledger::PublisherInfo());
```

Comment:
use `make_unique` here.

Reply:
what is the difference?

Reply:
You're resetting an already empty unique_ptr. In this case they are technically equivalent, but `make_unique` shows that you are initializing a new smart pointer rather than changing ownership.

Reply:
before it's not empty but null, with reset I set default values for `ledger::PublisherInfo` object to the existing pointer

Reply:
done